extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -571,8 +607,20 @@ export class AmpStoryPage extends AMP.BaseElement {
         : this.maybeStartAnimations_();
       this.checkPageHasAudio_();
       this.checkPageHasElementWithPlayback_();
-      this.renderOpenAttachmentUI_();
       this.findAndPrepareEmbeddedComponents_();
+
+      const attachmentHref = this.element
+        .querySelector('amp-story-page-attachment')
+        .getAttribute('href');
+
+      if (
```

Comment:
Can we avoid mixing `&&`'s and `||`'s in the same condition, and test positive assertions where possible?  Maybe:

```javascript
if (this.isPageAttachmentOutlinkV2ExperimentOn_() && attachmentHref) {
  this.renderOpenOutlinkAttachmentUI_();
} else {
  this.renderOpenAttachmentUI_();
}
```